torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -446,7 +556,8 @@ ComputeAt::ComputeAt(
       consumer_(_consumer),
       reference_(_reference),
       reference_position_(_reference_position),
-      mode_(_mode) {
+      mode_(_mode),
+      pred_map_(FusionGuard::getCurFusion()) {
```

Comment:
Is it safe to use a `ThreadPredicateMap` created here? At this point, if parallelization is not yet done, it won't detect block broadcast. Are we assuming `computeAt` comes after parallelization?

Reply:
ThreadPredicateMap was completely removed thanks to barrier syncs.